### DISCUSSION TITLE
Welcome to Arduino_uSDX_Pico_FFT_Proj Discussions!

### DISCUSSION THREAD

**Discussion by kaefe64** [kaefe64/Arduino_uSDX_Pico_FFT_Proj#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wheller3** [comment]
I am using your code and a  QSD/QSE external board however the main tuning encoder jumps frequency in both directions when I rotate the control slowly in either direction. Has anyone sen this issue and offer suggestions to fix this so I get smooth tuning.

**Reply by kaefe64** [reply]
Hi Wheller3

Look the "**Last changes and notes:**" on https://github.com/kaefe64/Arduino_uSDX_Pico_FFT_Proj
on the date  **Ago07 2022**

There are some comments about the encoder, and the option on the code. Look for ENCODER_TYPE on hmi.cpp.

**Comment by renato-IU0** [comment]
Hello and Happy New Year to everyone
My name is Renato and I'm IU0MTL in Rome. I'm a keen DIYer and recently built a USDX and Dawson Jon's Pi-Pico receiver. I'd love to stay in touch and share my doubts, uncertainties, or design ineptitudes, and seek advice where possible. Thanks from Renato.

**Reply by renato-IU0** [reply]
I'm clearly very interested in the ARJAN-5 project. So far, thanks to the precise instructions, I've managed to compile it and have problem-free control of both the encoder and the buttons, and the correct display on the ILI9341. I still need to assemble the rest of the hardware, and I'd like to share opinions on some hardware points.

**Comment by jasonmarkham** [comment]
I am looking at usdx pico fft trx project- seems there is no info on how the bn43-2402  trafo is wired from the rx qsd and tx qse . is this documented somewhere ?  and is there a full connection diagram ? 

73's
Jason 
9H5BM
